### PR TITLE
[WIP] tests, storage: Fail if block storage not present

### PR DESCRIPTION
**Pull request by RamLavi**
WIP

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
phase 1: checking which tests fail when there is no skip pattern.
/test pull-kubevirt-e2e-k8s-1.29-sig-storage
/test pull-kubevirt-e2e-k8s-1.30-sig-storage
/test pull-kubevirt-e2e-k8s-1.31-sig-storage
/test pull-kubevirt-e2e-k8s-1.31-sig-compute-migrations

phase 2: Set the decorators in lanes where needed (need to check if that is done on prow/kubvirt repos).

File: tests/decorators/decorators.go [hunk 1]
```diff
@@ -17,15 +17,16 @@ var (
...
-	Multus      = Label("Multus")
-	Macvtap     = Label("Macvtap")
-	Invtsc      = Label("Invtsc")
-	KSMRequired = Label("KSM-required")
+	GPU                  = Label("GPU")
+	VGPU                 = Label("VGPU")
+	SEV                  = Label("SEV")
+	SRIOV                = Label("SRIOV")
+	StorageReq           = Label("storage-req")
+	BlockStorageRequired = Label("block-storage-required")
```
Comment (akalenyu) [inline tests/decorators/decorators.go:25]: I added decorators for this the other day - https://github.com/kubevirt/kubevirt/pull/13137

Comment (RamLavi) [reply tests/decorators/decorators.go:25]: thanks @akalenyu for being one step in front of me :) will close this PR then

**Pull request closed without merging**